meta-mender-core/recipes-mender/mender-client/mender-client.inc

```diff
@@ -1,7 +1,14 @@
 DESCRIPTION = "Mender tool for doing OTA software updates."
 HOMEPAGE = "https://mender.io"
 
+BBCLASSEXTEND = "native"
+RDEPENDS_${PN}_class-native = ""
+DEPENDS_class-native = ""
+
+PACKAGES_append = " ${PN}-modules-gen"
```

Comment:
AFAIK, it's not required to use `${PN}` here, so you could call it mender-modules-gen. I think that name is a bit better, but no strong preference.

Reply:
Thanks, changed. I also think is a better name.